modin/experimental/core/execution/native/implementations/omnisci_on_native/test/test_dataframe.py

```diff
@@ -283,7 +283,8 @@ def test_read_csv_datetime(
     ):
 
         parse_dates_unsupported = isinstance(parse_dates, dict) or (
-            isinstance(parse_dates, list) and isinstance(parse_dates[0], list)
+            isinstance(parse_dates, list)
+            and all(isinstance(date, list) for date in parse_dates)
```

Comment:
I don't think the test needs to be added here right? Since we control the values for `parse_dates` and we don't pass an empty list to it.

Reply:
Agreed. I reverted this change.

Reply:
We decided offline to test an empty list in omnisci, but that required re-added the check here, so I did that.